Cabal-tests/tests/ParserTests/regressions/all-upper-bound.check

```diff
@@ -0,0 +1,8 @@
+No 'maintainer' field.
```

Comment:
Nitpick: Add a `maintainer` and `description` in the cabal file. This way if description error changes there is one less thing to modify in the future.

Reply:
I added the package and maintainer fields to the cabal file, thanks.